docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS           = -W -n
+SPHINXOPTS           = -W
```

Comment:
Sadly no, sphinx's classifcations of these warnings is very broad. The ones we care about, like the broken reference that you found to `pulp.app.fields.JSONField` get lumped together with ones we don't, like references to `djang.db.models.Model`.